app/controllers/DraftMetadataChecksResultsController.scala

```diff
@@ -37,17 +36,19 @@ class DraftMetadataChecksResultsController @Inject() (
     } else {
       val token = request.token.bearerAccessToken
       for {
-        reference <- consignmentService.getConsignmentRef(consignmentId, request.token.bearerAccessToken)
-        consignmentStatuses <- consignmentStatusService.getConsignmentStatuses(consignmentId, token)
-        draftMetadataStatus = consignmentStatuses.find(_.statusType == DraftMetadataType.id).map(_.value)
+        consignmentDetails <- consignmentService.getConsignmentDetails(consignmentId, token)
+        reference = consignmentDetails.consignmentReference
+        statuses = consignmentDetails.consignmentStatuses
+        uploadedFileName = consignmentDetails.clientSideDraftMetadataFileName.getOrElse("Not Available")
```

Comment:
The call is unlikely to fail but if it did "Not Available" will be displayed in the UI which might be confusing to the user. Might be best not to show it at all but that's another discussion.